common/actions.js

```diff
@@ -1,6 +1,8 @@
 import LocalStorage from '../utils/LocalStorage'
+import { buildIntentsDataFromCyElements } from '../cy_canvas/helper'
 
 let intentId = parseInt(LocalStorage.getElements("elementId"));
+const devKey = LocalStorage.getDevKey();
```

Comment:
It will be assigned only once when the page launched. So the first time user open this page, there won't be any value even after it is filled and saved.